spec/bill_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'bill'
+
+describe Bill do
+
+  # let(:menu) { double :menu, :menu_items[:pie] => 4.5, :menu_items[:cranberry_juice] => 2.50 }
+  let(:get_orders) { { :pie => 2, :cranberry_juice => 2 } }
+  let(:menu) { { :pie => 4.5, :cranberry_juice => 2.5 } }
+
+  let(:conf) { double :conf }
+
+  it "returns the bill" do
+    # get_orders = { :pie => 2, :cranberry_juice => 2 }
+    cust_bill = Bill.new(get_orders, menu)
+    expect(cust_bill.bill).to eq 14
+  end
+
+  it 'settles the bill' do
+    # get_orders = { :pie => 2, :cranberry_juice => 2 }
+    cust_bill = Bill.new(get_orders, menu)
+    cust_bill.bill
+    expect { cust_bill.conf_message(0, conf) }.to raise_error "please settle the bill"
```

Comment:
This test is a little unclear to me.  It looks as though your raising an error because there's nothing on the bill, but thats not what the error message says.

The naming of the test is also a little confusing - it doesn't seem to match up to what the test is actually doing.